packages/core/src/client.ts

```diff
@@ -320,6 +320,10 @@ export interface MedplumRequestOptions extends RequestInit {
    * Default value is 1000 (1 second).
    */
   pollStatusPeriod?: number;
+  /**
+   * Optional max number of retries that should be made in the case of a failed request. Default is `2`.
+   */
+  maxRetries?: number;
```

Comment:
Users might also want to customize the retry delay or other characteristics of the retry loop; one possible way we could accommodate this would be to allow them to pass a function option like this:
```ts
retryStrategy?: (url: string, retryCount: number, response: Response) => number | undefined;
```

Reply:
I think it's probably a good idea to have this at some point, want to open an MQ issue?